### DISCUSSION TITLE
czg配置文件是否可以设置在项目目录外（process.cwd()外）?

### DISCUSSION THREAD

**Discussion by Bellyang** [Zhengqbbb/cz-git#207] [Q&A]
使用--config配置目录地址并且指向项目外的文件，查看源码中的configLoader可以加载配置文件，但是不生效（同样文件放根目录是生效的），请问是否可以支持配置在任意位置的设置？

**Comment by Zhengqbbb** [comment]
可以的呀，有没有例子提供一下
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/fc319bb0-2384-41d5-b216-82513445cdca">

**Comment by Bellyang** [comment]
我再debug了下，发现config文件不能是
`{
  rules: {},
  prompt: {}
 }
`
这样的，要直接就是
`
{
 // prompt
 alias: { fd: 'docs: fix typos' }
}
`
理解错了，抱歉，不过好像放根目录的话上面那种格式也没问题

**Comment by Zhengqbbb** [comment]
是的 除非是 commitlint 的配置才需要添加 `prompt` 字段包裹，但 `--config` 仅支持 cz 配置足以
